lib/ndr_import/file/vcf.rb

```diff
@@ -8,8 +8,33 @@ module NdrImport
   module File
     # This class is a vcf file handler that returns a single table.
     class Vcf < Base
+      attr_accessor :vcf_file_metadata
+
+      def initialize(*)
+        super
+
+        @vcf_file_metadata = @options['vcf_file_metadata']
+        assign_file_metadata
+      end
+
       private
 
+      def assign_file_metadata
+        return unless vcf_file_metadata.is_a?(Hash)
+
+        file_metadata_hash = {}
+
+        ::File.read(@filename).each_line do |line|
+          next unless line =~ /^##/
+
+          vcf_file_metadata.each do |attribute, pattern|
+            file_metadata_hash[attribute] = line.match(pattern)[1].presence if line =~ pattern
```

Comment:
match? unless you don't think the regex security concern can be triggered here

Reply:
`match?` will also convert `pattern` to a Regexp